roles/janus/defaults/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+upstream_name:
```

Comment:
maybe add some default value like "<my-namespace-here>" and "<my-upstream-name-here>", which would ensure it fails with a clear(er) error message?

Reply:
not really needed at this point, since the specs has required: True and not defining the var will the role invocation fail before it is executed